src/main/java/se/kth/meta/wscomm/WebSocketEndpoint.java

```diff
@@ -71,7 +70,7 @@ public void open(Session session, EndpointConfig config,
 
     //returns the user role in project. Null if the user has no role in project
     this.userRole = projectTeamBean.findCurrentRole(this.project, this.sender);
-    logger.log(Level.SEVERE, "User role in this projuct {0}", this.userRole);
+    logger.log(Level.SEVERE, "User role in this product {0}", this.userRole);
```

Comment:
This sounds more like debug information than a SEVERE warning? (If so, adapt the logging level.)